docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -299,7 +299,7 @@ There are several types of content in collections which licensing has to address
 * The content that MUST be licensed with a free software license that is **compatible with** the `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_:
 
   * The ``modules/`` directory content.
-  * The ``module_utils/`` directory content: ansible-core itself typically uses the `BSD-2-clause <https://opensource.org/licenses/BSD-2-Clause>`_ license to make it possible for third-party modules which are licensed incompatibly with the GPLv3 to use them.Please consider this use case when licensing your own ``module_utils``.
+  * The ``module_utils/`` directory content: ansible-core itself typically uses the `BSD-2-clause <https://opensource.org/licenses/BSD-2-Clause>`_ license to make it possible for third-party modules which are licensed incompatibly with the GPLv3 to use them. Please consider this use case when licensing your own ``module_utils``.
```

Comment:
```suggestion
  * The ``module_utils/`` directory content: ansible-core typically uses the `BSD-2-clause <https://opensource.org/licenses/BSD-2-Clause>`_ license to allow third-party modules to use the ``module_utils`` in cases when those third-party modules have licenses that are incompatible with the GPLv3. Please consider this use case when licensing your own ``module_utils``.
```

Reply:
This revision has a higher word count and is a bit repetitious but it took me a couple reads to grok the original.

Reply:
wfm, committed, thanks, PTAL